hardware/edid.ksy

```diff
@@ -257,6 +267,195 @@ types:
         1: ratio_4_3
         2: ratio_5_4
         3: ratio_16_9
+  dtd_timing:
+    seq:
+      - id: pixel_clock_mod
+        type: u2
+        doc: |
+          Pixel Clock / 10,000
+      - id: horiz_active_pixels_lo
+        type: u1
+      - id: horiz_blanking_lo
+        type: u1
+      - id: horiz_active_pixels_hi
+        type: b4
```

Comment:
I think in this specific case, it would be wise to indicate the bit width of each of these "partial fields" in the identifier, for example:

```suggestion
      - id: horiz_blanking_lo_b8
        type: u1
      - id: horiz_active_pixels_hi_b4
        type: b4
```

The idea behind it is to express "the lower 8 bits of `horiz_blanking`" and "the upper 4 bits of `horiz_active_pixels`". I think this will make the manual bitwise operations easier to check, because you'll be able to see whether the bit shift amount agrees with the bit width of the lower operand, for example this...:

https://github.com/kaitai-io/kaitai_struct_formats/blob/36c7af9da09f62b85d8a9a93cf0d93bb06cca93a/hardware/edid.ksy#L339-L340

... becomes this:

```ksy
      horiz_active_pixels:
        value: horiz_blanking_lo_b8 | (horiz_active_pixels_hi_b4 << 8)
```

Or actually, it would be probably better if you swap the OR operands:

```ksy
      horiz_active_pixels:
        value: (horiz_active_pixels_hi_b4 << 8) | horiz_blanking_lo_b8
```

---

And this...:

https://github.com/kaitai-io/kaitai_struct_formats/blob/36c7af9da09f62b85d8a9a93cf0d93bb06cca93a/hardware/edid.ksy#L374-L375

... would be:

```ksy
      vert_sync_pulse:
        value: (vert_sync_pulse_hi_b2 << 4) | vert_sync_pulse_lo_b4
```

Reply:
Fixed in https://github.com/N-Nagorny/kaitai_struct_formats/commit/e1edfef38739c85c8ea636a265dea8784f4e6490